dev/1_load_utility.ipynb

```diff
@@ -0,0 +1,240 @@
+{
```

Comment:
I'd recommend you to click on "View file" to see the rendered version of the ipynb, if it does not show up by default!